neo/d3xp/Game_local.cpp

```diff
@@ -604,6 +605,8 @@ void idGameLocal::SaveGame( idFile *f ) {
 	savegame.WriteBool( isMultiplayer );
 	savegame.WriteInt( gameType );
 
+	savegame.WriteFloat(preciseTime);
```

Comment:
this breaks savegame compatibility :-(

Reply:
Thanks for letting me know this. I will try to find a way to avoid that, in fact I was looking at the BFG Edition code like a lot, and there are many things that I don't like about this pull request after looking at that. So I will try to find a way to improve this or actually try this again but most similar to the BFG Edition approach.

Reply:
I *think* the best way to do this is to bump `BUILD_NUMBER` and in `idGameLocal::InitFromSaveGame()` you can check `savegame.GetBuildNumber()` and only if it's the new number you read preciseTime, otherwise you set preciseTime to some sane default value.